### ISSUE TITLE
Drag and drop doesn't work

### ISSUE THREAD

**Issue by moduld** [georgipeltekov/ngx-file-drop#219]
The component works fine when I attach files by click, but when I try to add a file by drag-and-drop - it emits array of wrong entities.
Also it duplicates the emitted data (if I add one file - emitted data will containe 2 items)
![image](https://user-images.githubusercontent.com/17098835/101792319-3ed9b200-3b0d-11eb-9759-38f1a9009c2e.png)

On the screenshot result of adding one image.

Env: Angular 10, Chrome 87.0.4280.88, Linux

**Comment by georgipeltekov** [comment]
@moduld  Could you please give some stackblitz example, where it is reproducible

**Comment by moduld** [comment]
@georgipeltekov  When I was creating stackblitz I realised that the module actually works. The issue was on my side. I implemented the logic
...
const fileEntry = droppedFile.fileEntry as FileSystemFileEntry;
fileEntry.file((file: File) => {
...

in different way, but looks like it should be exactly as it is in the module example.
After I reworked my code - it started to work as expected. 

Thank your for the module!
